remote_server/remote_server_installer.sh

```diff
@@ -0,0 +1,533 @@
+#!/bin/sh
+
+BASE_PATH='/usr/local/copperegg/ucm-metrics'
+
+setup_base_group()
+{
+    TYPE_LOWER="$1"
+    TYPE_UPPER="$2"
+
+    echo
+    echo "Configuring $TYPE_UPPER"
+
+    echo -n "group_name: [$TYPE_LOWER] "
+    read GROUP_NAME
+    if [ -z "$GROUP_NAME" ]; then
+        GROUP_NAME="$TYPE_LOWER"
+    fi
+
+    echo -n "group_label: [$TYPE_UPPER Metrics] "
+    read GROUP_LABEL
+    if [ -z "$GROUP_LABEL" ]; then
+        GROUP_LABEL="$TYPE_UPPER Metrics"
+    fi
+
+    echo -n "dashboard: [$TYPE_UPPER] "
+    read DASHBOARD
+    if [ -z "$DASHBOARD" ]; then
+        DASHBOARD="$TYPE_UPPER"
+    fi
+
+    echo "$TYPE_LOWER:" >> $CONFIG_FILE
+    echo "  group_name: \"$GROUP_NAME\"" >> $CONFIG_FILE
+    echo "  group_label: \"$GROUP_LABEL\"" >> $CONFIG_FILE
+    echo "  dashboard: \"$DASHBOARD\"" >> $CONFIG_FILE
+    echo "  servers:" >> $CONFIG_FILE
+}
+
+setup_remote_server()
+{
+    COUNTER=$1
+    DEFAULT_PORT=22
+    DEFAULT_USER='ubuntu'
+
+    echo
+    echo
+    echo -n "Unique_id: "
+    read LABEL
+    if [ -z "$LABEL" ]; then
+        echo "Unique_id cannot be blank"
+        return 1
+    fi
+
+    echo
+    echo
+    echo -n "Server URL or IP (without HTTP or HTTPS) "
+    read URL
+    if [ -z "$URL" ]; then
+        echo "Hostname cannot be blank"
+        return 1
+    fi
+
+    echo
+    echo "User [ubuntu] "
+    echo "Hint : When you do ssh, you write username@servername.com, write that 'username' here "
+    read USER
+    if [ -z "$USER" ]; then
+        USER="$DEFAULT_USER"
+    fi
+
+    echo
+    echo "Password (if any) "
+    echo "Hint : Did you set any password at the time of generating ssh keys ? If yes, add it here otherwise hit Enter "
+    read PASSWORD
+
+    echo
+    echo -n "Port: [22] "
+    read PORT
+    if [ -z "$PORT" ]; then
+        PORT="$DEFAULT_PORT"
+    fi
+
+    echo
+    echo "SSH Keys (public / private key pair) :"
+    echo "Hint : Must give a key pair path, so that if the program runs with $COPPEREGG_USER user, it can ssh sucessfully "
```

Comment:
added 
echo "This should be an absolute path like /home/user/.ssh/some_rsa_or_pem_file"